Firestore/core/src/firebase/firestore/local/leveldb_query_cache.mm

```diff
@@ -46,7 +47,7 @@
 FSTPBTargetGlobal* LevelDbQueryCache::ReadMetadata(leveldb::DB* db) {
   std::string key = LevelDbTargetGlobalKey::Key();
   std::string value;
-  Status status = db->Get([FSTLevelDB standardReadOptions], key, &value);
+  Status status = db->Get(StandardReadOptions(), key, &value);
```

Comment:
Question: is this a free function now?

Reply:
Yes, though in the `firebase::firestore::local` namespace, so unlikely to collide with anything else.